### PR TITLE
Update documentation for config.kueue.x-k8s.io v1beta1 

### PR DESCRIPTION
**Pull request by tenzen-y**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
I updated documentation for `config.kueue.x-k8s.io` and `kueue.x-k8s.io` v1beta1.

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Read the [overview](https://kueue.sigs.k8s.io/docs/overview/) to learn more.
 - ✔️ Monitoring via [metrics](https://kueue.sigs.k8s.io/docs/reference/metrics)
 - ✔️ API validation and defaulting.
 - ⏳ Scalability tests in progress.
-- ⚠️ Latest API version: v1alpha2.
+- ⚠️ Latest API version: v1beta1.
```
[the remark this answers is not in the document]
Comment (tenzen-y) [reply README.md]: Done.

**Comment by tenzen-y** [comment]
The netlify failed to enqueue a building job.

> 4:02:15 AM: Error enqueueing build.

**Comment by tenzen-y** [comment]
I'll do a force push to re-kick a building job.

**Review by alculquicondor — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**